src/entry-client.js

```diff
@@ -129,7 +134,7 @@ export function useRouteVuexModulesClient(app, router, store, logger) {
     router.beforeResolve((to, from, next) => {
         try {
             const fetchDataArgs = getFetchDataArgs(null, app, router, store, to, from);
-            router.getMatchedComponents(to)
```

Comment:
Built-in method removed - https://next.router.vuejs.org/guide/migration/#removal-of-router-getmatchedcomponents